MekHQ/data/universe/news.xml

```diff
@@ -22,1912 +22,1912 @@ location - an optional tag for the location of the news report
-		<date>2825</date>
-		<desc><![CDATA[<p>The most recent casualty of the Succession Wars has just made its mark on the planet Lone Star. The moon, Vespa, entered the atmosphere earlier this week with devastating results. While it is unclear at this time how many injuries there are, this event will change the face of the planet for generations to come.</p>]]></desc>
-		<service>SBC</service>
-		<location>Lone Star</location>
-	</newsItem> 
-	<newsItem><!-- ISSUE - Rewording required -->
-		<headline>Marik forces destroyed after fleeing Solaris</headline><!-- Source: Solaris: The Reaches -->
-		<date>2903-05</date>
-		<desc><![CDATA[<p>Filed 02 April 2903</p><p>There is muted celebration on Solaris VII today as the war criminals from the Free Worlds League were immediately and irrevocably destroyed during a failed jump out of the system. The ship, <em>Despiser</em>, opened fire on the world with nuclear weapons before attempting to flee. A failed jump resulted in the destroyer breaking up above the planet in an incredible display of light. While there is no current accounting of casualties, the areas around Xolara are currently being treated as a hazardous zone and all emergency personnel responding to the incident are required to have the maximum level of protective gear.</p>]]></desc>
-		<service>CBN</service>
-		<location>Solaris VII</location>
-	</newsItem>
-	<newsItem>
-		<headline>Ruins to hold classes once again</headline><!-- Source: Field Manual: Draconis Combine -->
-		<date>2910</date>
-		<desc><![CDATA[<p>Reconstruction has completed on a facility almost entirely destroyed in 2865. The rebuilt facility, now named the Dover Institute of Higher Learning, is slated to begin its first classes later this year. The Dover Institute of Higher Learning is to be a centre of learning and training to ensure the Combine is ready for the years to come, home to a technician school, an engineering school, a medical school, and even a small but elite MechWarrior training program. The curriculum is anticipated to be extremely intense, to produce elite graduates for the glory of the Draconis Combine.</p>]]></desc>
-		<service>CBN</service>
-		<location>Ashio</location>
-	</newsItem>
-	<newsItem>
-		<headline>Parliament seeks restitution from First Lord and Captain-General</headline><!-- Source: House Marik (The Free Worlds League) -->
-		<date>2989-10</date>
-		<desc><![CDATA[<p>Filed 12 September 2989</p><p>Parliament is setting a new precident by demanding restitution from the First Lord and Captain-General and the military as a whole to replenish funds used during the recent wars. Special Resolution 512 calls for funds, materials, and equipment to be disbursed back into the FWL economy to help bolster those industries most effected by the fighting. No official word from the office of the First Lord. Economists are following this news intently as these reparations could significantly bolster struggling economies across the FWL.</p>]]></desc>
```

Comment:
precedent I believe vs establishing a new ruling member

Reply:
Both are mistakes... it should be precedent as you stated.